scapy/layers/bluetooth.py

```diff
@@ -210,6 +211,54 @@ class BT_Mon_Pcap_Hdr(BT_Mon_Hdr):
     ]
 
 
+class BT_Mon_New_Index(Packet):
```

Comment:
Could you actually keep the `HCI_Mon`... name (to be consistent with other HCI_ packets).
It might also be nice to rename `BT_Mon_Hdr` into `HCI_Mon_Hdr`.
Thanks

Reply:
Well... Monitor socket protocol is not HCI... It can contain HCI packets in lower layers but it also contains a lot of packets that are not HCI related but only related to BlueZ...

I personally don't see the logic behind renaming this layer to HCI_Mon as it could lead to confusion. The protocol is only specified by BlueZ and not Bluetooth SIG nor the Bluetooth core specification.

Do you still prefer to rename it?

Thanks

Reply:
Wireshark calls this [`hci_mon`](https://wiki.wireshark.org/Bluetooth#supported-profiles-and-protocols) and the socket type is `HCI_CHANNEL_MONITOR`. (right?) I'd argue it would be better to keep some level of consistency with what other tools/programms use.
Thoughts?

Reply:
Yes, it is confusing...
Wireshark calls this `hci_mon` but the full protocol name is `Bluetooth Linux Monitor Transport` and the proto is not described in Bluetooth Core HCI sections. Wireshark also defines `hci_h1`, `hci_h4` and `hci_usb` wich are described as `Bluetooth HCI ... Transport`.

If we rely on BlueZ implementation to settle the doubts it is even worse...
The socket type is `HCI_CHANNEL_MONITOR`. I am not sure why BlueZ developers did this... In my opinion, that type should be reserved for HCI protocol packets and the "HCI Monitor" protocol is not HCI but a proto that can encapsulate HCI.

On the other hand, looking at BlueZ code, examples of hdr struct for this proto do not contain the HCI keyword: https://github.com/bluez/bluez/blob/6a8f7c55dfdfed8136ba8b417d0d8ee45181001a/tools/btmon-logger.c#L41

Anyway, as said in the previous paragraph, if we understand that the protol is intended to be an `HCI Monitor` protocol that is enough I guess. My only worry about renaming is that when packets will be named `HCI_Mon_...` it will look like those are HCI packets or subprotocols when they are not...

I will soon rename the layer to be HCI_Mon :)

Thanks